src/Features/Core/Portable/PreferFrameworkType/PreferFrameworkTypeDiagnosticAnalyzerBase.cs

```diff
@@ -50,18 +51,30 @@ public override DiagnosticAnalyzerCategory GetAnalyzerCategory()
         protected sealed override void InitializeWorker(AnalysisContext context)
             => context.RegisterSyntaxNodeAction(AnalyzeNode, SyntaxKindsOfInterest);
 
+        void Test()
```

Comment:
Revert?

Reply:
This is why I need a 3rd Roslyn enlistment, so I don't test things in the same place I am making changes. Will revert.